eng/tools/rush-runner.js

```diff
@@ -78,7 +77,7 @@ const getServicePackages = (baseDir, serviceDirs) => {
     const packageJsons = getPackageJsons(searchDir);
     for (const filePath of packageJsons) {
       const contents = JSON.parse(fs.readFileSync(filePath, "utf8"));
-      if (contents["sdk-type"] === "client") {
+      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "management") {
```

Comment:
```suggestion
      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "mgmt") {
```

Most other places call this mgmt so lets do that here as well.

Reply:
@sarangan12 just made changes to the code generator to use `management`. 
@praveenkuttappan Please confirm which way we want to go here so that @sarangan12 can make the changes appropriately

Reply:
@praveenkuttappan I prefer `mgmt`, which is used in most places for management plane.

Reply:
as the codegen is using `management` now, we will use it in pipeline too.

Reply:
I will change my PR to look for mgmt. @sarangan12 : Can you please change it to mgmt in code gen.